packages/context/src/binding.ts

```diff
@@ -131,10 +135,27 @@ export class Binding {
     };
   }
 
+  /**
+   * Get nested properties by path
+   * @param value Value of an object
+   * @param path Path to the property
+   */
+  static getDeepProperty(value: BoundValue, path: string) {
+    const props = path.split('.').filter(Boolean);
```

Comment:
In ES2015, it's common to pass `undefined` for arguments that we don't want to specify. This plays well with default argument values too.

I am proposing to change `getDeepProperty` to support `path` value `undefined` too.